app/views/past_foreign_secretaries/george_curzon.html.erb

```diff
@@ -1,47 +1,44 @@
 <% page_title "History of George Nathaniel Curzon" %>
-<% page_class "historic-people-show" %>
+<% page_class "historic-people-show govuk-width-container" %>
```

Comment:
I wonder what the reason is for all of these pages being hard coded, and not a single template that content is passed to?

Reply:
I don't know why - it's hopefully going to be looked at when removing the pages them from Whitehall though.